http-server/apache2-web-server/README.md

```diff
@@ -0,0 +1,132 @@
+# HTTP Server
+
+In this example, we will run a simple Apache Web server in a Singularity Container.
+
+<br>
+
+What you need:
+ - Singularity, you can download and install it [here](https://github.com/sylabs/singularity).
+ - A text editor, like: `micro`, `vim` or `nano`.
+ - root access.
+ - a browser to test it, or you can use `w3m`, install it by `sudo apt-get install w3m w3m-img`.
+ 
+
+<br>
+
+To start, make the working directory:
+
+```
+mkdir httpd
+cd httpd/
+```
+
+<br>
+
+Then, make the definition file:
+```
+micro httpd.def
```

Comment:
Change to:  `micro` to `nano`

nano is one that is now usually installed.... *sigh*